orange/src/test/java/com/hartwig/hmftools/orange/algo/cuppa/CuppaDataFactoryTest.java

```diff
@@ -58,11 +58,12 @@ public void canExtractPredictionsFromCuppaIncludingRna() throws Exception
                 .build();
 
         Map<String, CuppaPrediction> expectedPredictionsByCancerType = new HashMap<>();
-        expectedPredictionsByCancerType.put("Skin: Melanoma", expectedPredictionMelanoma);
-        expectedPredictionsByCancerType.put("Skin: Other", expectedPredictionSkinOther);
-        expectedPredictionsByCancerType.put("Prostate", expectedPredictionProstate);
+        expectedPredictionsByCancerType.put(expectedPredictionMelanoma.cancerType(), expectedPredictionMelanoma);
+        expectedPredictionsByCancerType.put(expectedPredictionSkinOther.cancerType(), expectedPredictionSkinOther);
+        expectedPredictionsByCancerType.put(expectedPredictionProstate.cancerType(), expectedPredictionProstate);
+        expectedPredictionsByCancerType.put("Bone/Soft tissue: Cartilaginous neoplasm", null);
```

Comment:
Where is the null setup in the test data? ... oh ok, I see that it was always there but its actually missing values in the input file from CUPPA and not explicit NaN's.

Reply:
Correct, that's what tricked us in the first place :)